test/functional/ui/multibyte_spec.lua

```diff
@@ -465,9 +465,9 @@ describe('multibyte rendering: statusline', function()
       grid = [[
       ^                                        |
       {1:~                                       }|
-      {3:🧑�💻                                   }|
```

Comment:
I'm kind of confused why this test exists lol :3 . A few line below it also tests `🧑<200b>💻` is shown when custom statusline is used. Feels like we were testing a bug.